### DISCUSSION TITLE
Upload to Google Cloud

### DISCUSSION THREAD

**Discussion by jakubjanousek** [motor-admin/motor-admin#44] [Q&A] [answered]
Hi there,
First of all, thanks for a great product.

I wanted to ask if you thought about the best way to upload attachments to Google Cloud. Our use case is that we store images in GCP and only have URLs in our DB. Right now, the only way is to create a custom form & endpoint that would handle the editing & upload, right? 

Or is there a better way that I missed?

Thanks a lot for looking into this.

**Comment by omohokcoj** [comment, marked as answer]
@jakubjanousek upload to google cloud bucket has been added in the latest 0.2.65 release.
In order to upload  to GCS you need to add your google credentials on the settings>storage page.
You can learn about how to obtain those credentials/key here (see google storage part of the article):
https://hixonrails.com/ruby-on-rails-tutorials/ruby-on-rails-active-storage-aws-s3-google-cloud-platform-microsoft-azure/
